frontend/src/libs/packages/map/map-connector.package.ts

```diff
@@ -30,13 +31,16 @@ class MapConnector implements IMapConnector {
       const loader = new Loader({
         apiKey,
       });
-      const [GeocodingLibrary, RoutesLibrary] = await Promise.all([
-        loader.importLibrary('geocoding'),
-        loader.importLibrary('routes'),
-      ]);
+      const [GeocodingLibrary, RoutesLibrary, AutocompleLibrary] =
```

Comment:
I think PlacesLibrary is more appropriate here.
```suggestion
      const [GeocodingLibrary, RoutesLibrary, PlacesLibrary] =
```